microsoft-edge/index.yml

```diff
@@ -0,0 +1,445 @@
+### YamlMime:Hub
+
+title: Microsoft Edge developer documentation
+summary: Microsoft Edge is the browser built for Windows 10, providing fast, safe, and productive web experiences.  Here are all the developer features and tools available to you in Microsoft Edge and the Chromium engine.
```

Comment:
This feels out of date. We build on macOS and ARM now too. We should probably get one of the writers/branding folks to come up with the correct verbiage to put here.

Reply:
I'm suggesting for now we change it to "Explore documentation resources for Microsoft Edge" and then we can work with marketing after 1/15.

Reply:
Updated.
"Explore documentation resources for Microsoft Edge.  Here are all the developer features and tools available to you in Microsoft Edge and the Chromium engine."